base/build.gradle.kts

```diff
@@ -7,4 +12,15 @@ dependencies {
     }
 }
 
-version = property("baseApiVersion")!!
\ No newline at end of file
+version = property("baseApiVersion")!!
```

Comment:
Can we rename `baseApiVersion` to `version` in  `gradle.properties`? It's a remainder of when the Geyser api was also in this repo. Making it `version` will make this line unneeded.